### PR TITLE
[NONMODULAR] adds sleeping after head smashes

### PR DESCRIPTION
**Pull request by azrael-x86**
## About The Pull Request

allows you to put someone in a sleeping period of 50 (5 seconds) after you get your head smashed against a table.

unknown

(LIUVIK'S SEAL OF APPROVAL)

## How This Contributes To The Skyrat Roleplay Experience

when you get your head slammed against a table loud enough to make a large banging sound, you tend to go sleepy night night for a few seconds.

## Changelog

:cl:
expansion: Nanotrasen's Expert Combat Mechanics have finally realized you actually pass out for a few seconds after you get table slammed! Innovation, people.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/tables_racks.dm [hunk 1]
```diff
@@ -175,6 +175,7 @@
 		extra_wound = 20
 	banged_limb?.receive_damage(30, wound_bonus = extra_wound)
 	pushed_mob.apply_damage(60, STAMINA)
+	pushed_mob.SetSleeping(300) // SKYRAT EDIT
```
Comment (GoldenAlpharex) [inline code/game/objects/structures/tables_racks.dm]: 30 seconds of sleeping on top of all of that, for a move that takes about 10 seconds to do, probably even less? And that you can probably repeat once they're asleep? Sounds a bit overly busted.

File: modular_skyrat/modules/medical/code/wounds/bones.dm [hunk 2]
```diff
@@ -202,7 +202,7 @@
 	examine_desc = "is awkwardly jammed out of place"
 	occur_text = "jerks violently and becomes unseated"
 	severity = WOUND_SEVERITY_MODERATE
-	viable_zones = list(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
+	viable_zones = list(BODY_ZONE_HEAD, BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/medical/code/wounds/bones.dm]: I feel like dislocating your head would have much, much worse effects than regular Joint Dislocation.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I'm not sold on this one, chief.

**Comment by azrael-x86** [comment]
@GoldenAlpharex i pretty much expected a fair bit of adjustments would need to be done, but just wanted to give a baseline. will address these in a commit, one moment

**Pull request closed without merging**